modules/thunder_article/modules/thunder_fia/templates/user--facebook-instant-articles-rss.html.twig

```diff
@@ -0,0 +1 @@
+<a>{{ user.displayName }}</a>
```

Comment:
If we use a new viewmode for the user, shouldn't we then also render the full user?